src/EVM/Types.hs

```diff
@@ -1233,6 +1233,8 @@ abiKeccak =
 
 -- Utils -------------------------------------------------------------------------------------------
 
+internalError:: [Char] -> [Char]
+internalError a = "internalError " ++ a
```

Comment:
We could also make this function throw the error
``` error $ "Internal error:" <> a```
to avoid writing `error $ internalError` everywhere, since this is the only way it is used.

Reply:
Oh that's a good point :D

Reply:
Fixed! Thank you!